apps/frontend/app/(client)/_libs/apis/submission.ts

```diff
@@ -0,0 +1,47 @@
+import { safeFetcherWithAuth } from '@/libs/utils'
+import type { Submission, SubmissionDetail } from '@/types/type'
+import type { PaginationQueryParams } from './types'
+
+export interface GetSubmissionsRequest extends PaginationQueryParams {
```

Comment:
```suggestion
export interface GetSubmissionListRequest extends PaginationQueryParams {
```
list로 통일하면 좋을 것 같아요!